templates/test/ci/prow-azl3/patches/controller-manager.yaml

```diff
@@ -0,0 +1,46 @@
+- op: add
+  path: /spec/kubeadmConfigSpec/files/0
+  value:
+    content: |
+      #!/bin/bash
+
+      set -o nounset
+      set -o pipefail
+      set -o errexit
+
+      # Install ca-certificates packages for Azure Linux
+      tdnf install -y ca-certificates ca-certificates-legacy
```

Comment:
This setup looks reasonable to me. We should move some of this to the image-builder ansible scripts for Azure Linux--I'll create an issue or PR once this merges.

Reply:
Yeah I figured out most of these ips from trial and error. I tried using netstat but there were more ips I needed to enable from what that told me.